env0/common_template.go

```diff
@@ -0,0 +1,118 @@
+package env0
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type TemplateType string
+
+const (
+	CustomFlow     TemplateType = "custom flow"
```

Comment:
please make the types like we do in our own backend
`custom-flow`
`approval-policy`